clock.go

```diff
@@ -287,7 +287,7 @@ func (t *internalTicker) Next() time.Time { return t.next }
 func (t *internalTicker) Tick(now time.Time) {
 	select {
 	case t.c <- now:
-	case <-time.After(1 * time.Millisecond):
```

Comment:
I wasn't sure what motivated the change from `default` to a timeout here. The standard library uses a buffered channel with a default case (https://github.com/golang/go/blob/master/src/time/sleep.go#L94). I'm changing it to mirror the standard behavior more closely.